README.md
GlowTools
=========

changelog.txt
Changelog:
    
    v0.3a (1.5.1)
    + Dungeon/Pyramid/Mineshaft/Smithy/Stronghold loot!
    
    v0.3 (1.5.1)
    + Omni-Scepter :D
    + Configs :D
    = Changed the Block IDs around a bit; if you don't want all
      your existing GlowTools blocks to go missing, change them
      to what they were before (BEFORE you open your world!)
      Sorry for any inconveniences. :P
      Here's a list:            (Before, After)
            Block of Glowing Iron (244, 3000)
            Block of Red Iron (245, 3001)
            Glowing Sand (246, 3002)
            Candentinium Ore (247, 3003)
            Glowing Candentinium Ore (248, 3004)
            Glowing Glass (249, 3005)
            Glowing Sandstone (250, 3006)
            Block of Candentinium (251, 3007)
            Glowing Grass (252, 3008)
            Glowing Dirt (253, 3009)
            Glowing Cobblestone (254, 3010)
            Glowing Stone (255, 3011)
            Calcite (3000, 3012)
    = Glowing Grass now only spawns if the block below it is grass
      or dirt, and made rarer
    + Dark Door :O
    + Glowstone Infuser (Still a WIP)!
    + New Creative Tab!
    + Fancy Tooltip Stuff
    = Made Calcite veins less gigantic to avoid angering customers
    
    v0.2a (1.5)
    
    = Updated to 1.5!
    = Changed Cloak of Invisibility Texture
    = Recipe Stuff
    = Changed Cloning of Scepters of Light (now takes Candentinium
      instead of Diamond)
    + Smelting Candentinium Blocks gives 9 Candentinium Shards
    
    
    v0.2 (1.4.7)
    
    + Glowing Glass!
    + Glowing Sandstone!
    + Glowing Grass (cannot be crafted, but is spawned randomly
      on the surface [only spawns in 4% of chunks!])
    + Glowing Dirt!
    + Glowing Cobblestone!
    + Glowing Stone!
    + Glowing & Red Iron Nuggets! (try smelting your old glowing
      iron armor/glowing & red iron tools)
    + Calcite!
    + Cloak of Invisibility! (and some other stuff to make it)
    
    
    v0.1d (1.4.7)

    + Cloning of Scepters of Light at the cost of Block of Diamond
      (just place a Block of Diamond in the world and right-click
       on it with Scepter of Light in hand)
    + Scepter of Regeneration Mk. II (Regen 5...!)
    + Fashionable Stripey Glowing Iron Armor
    = Made Candentinium Ore Smeltable


    v0.1c (1.4.7)

    + Candentinium Ore & Shard
    = Changed Recipe for Scepter of Regeneration
    = Changed Recipe for Scepter Tip
    + This time the gfx file is actually included!


    v0.1b (1.4.7)

    + Scepter of Light - releases light
    + Scepter of Regeneration - heals you!
    + Crafting Recipes for Glowing & Red Iron Tools
    + Porkchops with Applesauce
    + Glowing Sand!


    v0.1a (1.4.7)

    = First release
    + Glowing Iron and Red Iron
    + Glowing Iron Block
    + Red Iron Block
    + Glowing & Red Iron Tools
    + Skidding Glow Arthropod
    

glowTools/ConfigLoader.java
package glowTools;

import net.minecraftforge.common.Configuration;
import cpw.mods.fml.common.event.FMLPreInitializationEvent;

public class ConfigLoader
{
	public static void loadConfig(FMLPreInitializationEvent event)
	{
		Configuration config = new Configuration(event.getSuggestedConfigurationFile());
		
		config.load();
		
		GlowTools.glowironBlockID = config.getBlock("Block of Glowing Iron ID", 3000).getInt();
		GlowTools.redironBlockID = config.getBlock("Block of Red Iron ID", 3001).getInt();
		GlowTools.glowsandBlockID = config.getBlock("Glowing Sand ID", 3002).getInt();
		GlowTools.CandentiniumOreID = config.getBlock("Candentinium Ore ID", 3003).getInt();
		GlowTools.CandentiniumOreGlowingID = config.getBlock("Glowing Candentinium Ore ID", 3004).getInt();
		GlowTools.GlowglassBlockID = config.getBlock("Glowing Glass ID", 3005).getInt();
		GlowTools.GlowSandStoneBlockID = config.getBlock("Glowing Sandstone ID", 3006).getInt();
		GlowTools.CandentiniumBlockID = config.getBlock("Block of Candentinium ID", 3007).getInt();
		GlowTools.GlowGrassBlockID = config.getBlock("Glowing Grass ID", 3008).getInt();
		GlowTools.GlowDirtBlockID = config.getBlock("Glowing Dirt ID", 3009).getInt();
		GlowTools.GlowCobbleBlockID = config.getBlock("Glowing Cobblestone ID", 3010).getInt();
		GlowTools.GlowingStoneBlockID = config.getBlock("Glowing Stone ID", 3011).getInt();
		GlowTools.FlowStoneBlockID = config.getBlock("Calcite ID", 3012).getInt();
		GlowTools.DarkDoorID = config.getBlock("Dark Door Block ID", 3013).getInt();
		GlowTools.GlowstoneInfuserID = config.getBlock("Glowstone Infuser ID", 3014).getInt();
		GlowTools.GlowstoneInfuserActiveID = config.getBlock("Active Glowstone Infuser ID", 3015).getInt();
		GlowTools.BlueGlassOreID = config.getBlock("Glassy Blue Ore ID", 3016).getInt();
		GlowTools.GlowFarmlandID = config.getBlock("Enriched Farmland ID", 3017).getInt();
		
		GlowTools.glowironItemID = config.getItem("Glowing Iron Ingot ID", 4200).getInt();
		GlowTools.glowmoltItemID = config.getItem("Melted Glowstone ID", 4201).getInt();
		GlowTools.redmoltItemID = config.getItem("Melted Redstone ID", 4202).getInt();
		GlowTools.glowredItemID = config.getItem("Blazing Blend ID", 4203).getInt();
		GlowTools.redironItemID = config.getItem("Red Iron Ingot ID", 4204).getInt();
		GlowTools.glironswordItemID = config.getItem("Glowing Iron Sword ID", 4205).getInt();
		GlowTools.glironpickItemID = config.getItem("Glowing Iron Pickaxe ID", 4206).getInt();
		GlowTools.glironshovelItemID = config.getItem("Glowing Iron Shovel ID", 4207).getInt();
		GlowTools.glironaxeItemID = config.getItem("Glowing Iron Axe ID", 4208).getInt();
		GlowTools.glironhoeItemID = config.getItem("Glowing Iron Hoe ID", 4209).getInt();
		GlowTools.redironswordItemID = config.getItem("Red Iron Sword ID", 4210).getInt();
		GlowTools.redironpickItemID = config.getItem("Red Iron Pickaxe ID", 4211).getInt();
		GlowTools.redironshovelItemID = config.getItem("Red Iron Shovel ID", 4212).getInt();
		GlowTools.redironaxeItemID = config.getItem("Red Iron Axe ID", 4213).getInt();
		GlowTools.redironhoeItemID = config.getItem("Red Iron Hoe ID", 4214).getInt();
		GlowTools.LightscepterItemID = config.getItem("Scepter of Light ID", 4215).getInt();
		GlowTools.SceptertipItemID = config.getItem("Scepter Tip ID", 4216).getInt();
		GlowTools.RedironhandleItemID = config.getItem("Red-Iron-Tool Handle ID", 4217).getInt();
		GlowTools.PorkappleItemID = config.getItem("Porkchop with Applesauce ID", 4218).getInt();
		GlowTools.RegenscepterItemID = config.getItem("Scepter of Regeneration ID", 4219).getInt();
		GlowTools.CandentiniumGemID = config.getItem("Candentinium Shard ID", 4220).getInt();
		GlowTools.GlironHelmID = config.getItem("Stripey Glowing Iron Helmet ID", 4221).getInt();
		GlowTools.GlironChestID = config.getItem("Stripey Glowing Iron Chestplate ID", 4222).getInt();
		GlowTools.GlironLegsID = config.getItem("Stripey Glowing Iron Leggings ID", 4223).getInt();
		GlowTools.GlironBootsID = config.getItem("Stripey Glowing Iron Boots ID", 4224).getInt();
		GlowTools.Regenscepter2ItemID = config.getItem("Scepter of Regeneration Mk. II ID", 4225).getInt();
		GlowTools.glowironNuggetID = config.getItem("Glowing Iron Nugget ID", 4226).getInt();
		GlowTools.redironNuggetID = config.getItem("Red Iron Nugget ID", 4227).getInt();
		GlowTools.pureCalciteItemID = config.getItem("Purified Calcite ID", 4228).getInt();
		GlowTools.cloakCoreID = config.getItem("Cloak Core ID", 4229).getInt();
		GlowTools.CloakInvisID = config.getItem("Cloak of Invisibility ID", 4230).getInt();
		GlowTools.ReflectorItemID = config.getItem("Light Reflector ID", 4231).getInt();
		GlowTools.OmniscepterItemID = config.getItem("Omni-Scepter ID", 4232).getInt();
		GlowTools.DarkDoorItemID = config.getItem("Dark Door ID", 4233).getInt();
		GlowTools.InfernalCoalItemID = config.getItem("Infernal Coal ID", 4234).getInt();
		GlowTools.BlueGlassShardID = config.getItem("Glassy Blue Shard ID", 4235).getInt();
		GlowTools.BlueFertilizerItemID = config.getItem("Blue Fertilizer ID", 4236).getInt();
		
		GlowTools.OmniscepterHardMode = config.get("Hard Mode", "Omniscepter Hard Mode", true).getBoolean(true);
		
		config.save();
	}
}
